packages/editor/src/client/locales/en.json

```diff
@@ -618,6 +620,7 @@
         "close": "Close",
         "confirm": "Confirm",
         "createdAt": "{{createdAt, ccc PP p}}",
+        "dateExplanationPopOver": "This message will explain precisely what the 'available online from' date does.",
```

Comment:
You still need to update this translation from pages. Otherwise looks great. :+1:

Reply:
Done 👍